docs/source/deployment/databricks/databricks_deployment_workflow.md

```diff
@@ -95,6 +95,10 @@ kedro new --starter=databricks-iris
 
 This command creates a new Kedro project using the `databricks-iris` starter template. Name your new project `iris-databricks` for consistency with the rest of this guide.
 
+ ```{note}
+ If you are not using the `databricks-iris` starter to create a Kedro project, then you should [disable file-based logging](../../logging/index.md#disable-file-based-logging) to prevent Kedro from attempting to write to the read-only file system.
+ ```
+
 ### Create an entry point for Databricks
```

Comment:
These are no longer valid because `logging.yml` is optional and disbaled by default. See https://github.com/kedro-org/kedro/issues/2671 @stichbury 
Should we reject this change or just address this in #2671?

Reply:
Ahhhh! I knew this was too simple. Because I was unaware, the change has already merged into `main` but is not published right now because it's only just gone in after 0.18.14. Anything here on `develop` will basically provide the source of truth. I suggest we continue and accept this change now but make a change for #2671 as you suggest to indicate that it applies to pre-0.19.0 only and revise copy further for logging changes in 0.19.0. 

But #2671 isn't a blocking change. So do you want to prioritise it or just move forwards with a note for now?

Reply:
Here on `develop` I have replaced the link to the logging docs to point to one in 0.18.14 because they've been reorganised for 0.19.0 and that section no longer exists.

I've had to use a full path to the HTML docs to point to a specific version. I couldn't work out at short notice if there was a relative link I could use that points just to a specific (and previous) version so I've hard-coded it but it doesn't sit that comfortably with me.

Happy for this to merge for now and we will address in later versions of the docs as described above. cc @astrojuanlu for